code/perception/src/lidar_distance.py

```
@@ -256,51 +254,82 @@
+                    np.ones(coordinates_xyz.shape[0]),
+                )
+            )
+        elif focus == "Right":
+            points = np.column_stack(
+                (
+                    coordinates_xyz[:, 0],
+                    coordinates_xyz[:, 2],
+                    coordinates_xyz[:, 1],
+                    np.ones(coordinates_xyz.shape[0]),
+                )
+            )
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)